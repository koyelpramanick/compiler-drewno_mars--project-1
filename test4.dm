1
test while1_
while
// testing out comments
