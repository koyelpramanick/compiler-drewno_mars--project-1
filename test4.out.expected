INTLITERAL:1 [1,1]
ID:test [2,1]
ID:while1_ [2,6]
WHILE [3,1]
EOF [5,1]
